packages/api/tests/api/gitHub.test.ts

```diff
@@ -1,28 +1,24 @@
 import "jest";
 import { gitHub  } from "../../src/api/auth/gitHub";
 import { testHandler } from "../testUtils/testHandler";
-import { env } from '../../src/env';
+import { env } from '../../src/__mocks__/env';
```

Comment:
You can get this from the same spot, since it'll be mocked. We need to use the original otherwise the actual code won't be using the same values

```suggestion
import { env } from '../../src/env';
```